src/ids-text/ids-text.scss

```
@@ -11,6 +11,18 @@
   &.audible {
     @include audible();
   }
+
+  &.ellipsis {
+    @include ellipsis;
```

```suggestion
    @include ellipsis;
```

Do these work? I thought it was @include ellipsis();

---

@tmcconechy yep. If there's no params for mixins, we can omit parens -- but for consistency sake I can change it.

---

Wish i knew that before. Well lets use () for now for consistency